quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow-integration-test/src/main/resources/greetUnknown.sw.json

```
@@ -0,0 +1,81 @@
+{
+  "id": "greetUnknown",
+  "version": "1.0",
+  "name": "Greeting workflow",
+  "expressionLang": "jsonpath",
+  "description": "JSON based greeting workflow",
+  "start": "ChooseOnLanguage",
+  "functions": [
+    {
+      "name": "greetFunction",
+      "type": "custom",
+      "operation": "sysout"
+    },
+    {
+       "name": "isEnglish",
+       "type": "expression",
+       "operation" : "$.[?(@.language  == 'English')]" 
+    }
+  ],
+  "states": [
+    {
+      "name": "ChooseOnLanguage",
+      "type": "switch",
+      "dataConditions": [
+        {
+          "condition": "fn:isEnglish",
+          "transition": "GreetInEnglish"
+        },
+        {
+          "condition": "${ $.[?(@.language  == 'Spanish')] }",
+          "transition": "GreetInSpanish"
+        }
+      ],
+      "defaultCondition": {
+        "transition": "NoGreet"
+      }
+    },
+    {
+      "name": "GreetInEnglish",
+      "type": "inject",
+      "data": {
+        "greeting": "Hello from JSON Workflow,"
+      },
+      "transition": "GreetPerson"
+    },
+    {
+      "name": "GreetInSpanish",
+      "type": "inject",
+      "data": {
+        "greeting": "Saludos desde JSON Workflow,"
+      },
+      "transition": "GreetPerson"
+    },
+    {
+      "name": "NoGreet",
+      "type": "inject",
+      "data": {
+        "greeting": "I'm not familiar with your language,"
+      },
+      "transition": "GreetPerson"
+    },
+    {
+      "name": "GreetPerson",
+      "type": "operation",
+      "actions": [
+        {
+          "name": "greetAction",
+          "functionRef": {
+            "refName": "greetFunction",
+            "arguments": {
+              "message": "$.greeting $.name"
+            }
+          }
+        }
+      ],
+      "end": {
+        "terminate": "true"
+      }
```

We don't have to set "terminate": true on end definition in SW unless we have "keepActive" top-level prop set to true.

```suggestion
      "end": true
```

---

Indeed, a copy paste thing, thanks for pointing out